### DISCUSSION TITLE
有api能获取当前可用节点数吗?

### DISCUSSION THREAD

**Discussion by svchostmm** [MetaCubeX/mihomo#1304] [Q&A]
目前运行方式是,./mihomo -f c.yaml
用的代理组,有8个订阅地址,一个地址有100-200左右的节点.加起来总共几千个节点,是否有一个办法查看检测后可用的节点呢?

**Comment by anaer** [comment]
http://127.0.0.1:9090/providers/proxies/SUB1
SUB1为代理组名, 可用节点判断 "alive": true
